apps/app/src/ui/dropdownLists/PartenairesDropdown/PartenairesDropdown.tsx

```diff
@@ -29,7 +30,10 @@ const PartenairesDropdown = (props: PartenairesDropdownProps) => {
   return (
     <SelectTags
       {...props}
-      placeholder={props.placeholder ?? 'Sélectionnez ou créez un partenaire'}
+      placeholder={
+        props.placeholder ??
+        `Sélectionner ${!props.disableEdition ? 'ou créer ' : ''}un partenaire`
+      }
```

Comment:
avec ce fonctionnement, on check donc à deux endroits que l'on peut éditer ce qui  n'est pas optimal:

1/ Directement dans `SelectTags` 
2/ mais également en props de tous les `xxxxDropdown`

Pour ne pas avoir de `props.disableEdition` partout, placeholder pourrait devenir une fonction 
 
```suggestion
};

const PartenairesDropdown = (props: PartenairesDropdownProps) => {
@@ -29,7 +30,10 @@ const PartenairesDropdown = (props: PartenairesDropdownProps) => {
return (
<SelectTags
{...props}
      placeholder={({isEditionDisabled}) => 
        props.placeholder ??
        `Sélectionner ${isEditionDisabled ? 'ou créer ' : ''}un partenaire`
      }
```

comme ça, toute la mécanique de savoir si un dropdown est éditable est au sein de `SelectTags`